frappe/core/doctype/rq_job/rq_job.py

```diff
@@ -227,6 +254,17 @@ def get_all_queued_jobs():
 	return [job for job in jobs if for_current_site(job)]
 
 
+def rq_comparator(job: frappe._dict, sort_key: str):
+	if (val := job[sort_key]) != _NOT_AVAILABLE:
+		return val
+	return -1
+
+
 @frappe.whitelist()
 def stop_job(job_id):
 	frappe.get_doc("RQ Job", job_id).stop_job()
+
+
+@frappe.whitelist()
+def requeue_job(job_id):
+	frappe.get_doc("RQ Job", job_id).requeue()
```

Comment:
I would rather just get rid of these by whitelisting the doc methods - but I'm not sure that would be the best idea _(I have not taken a closer look at how run_doc_method is executed lately)_

Reply:
This wont work because run_doc_method doesn't load doc from DB. It just inits it with dict received from client.